.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "endOfLine": "lf",
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "all",
+  "printWidth": 100
+}
```

Comment:
What for since we have prettier setting in eslint https://github.com/cloudquery/plugin-config-ui-lib/blob/main/.eslintrc.js?

Reply:
@jackcloudquery I believe if someone's IDE prettier settings differ from what our prettier settings are inside the eslintrc, then there is a conflict and the IDE wins - which breaks linting. By additionally having the prettierrc in the codebase it enforces these rules that match our linting. Does that track for you?

Reply:
okay but then I think it's worth doing the following:
1. Remove prettier configuration from eslint:
```
{
        endOfLine: 'lf',
        semi: true,
        singleQuote: true,
        tabWidth: 2,
        trailingComma: 'all',
        printWidth: 100,
      },
```
2. Convert `.prettierrc` to `.prettierrc.js` and export it from the project. This way we can still maintain the same style across all our plugins and still leave an option for customization if necessary